README.md

```diff
@@ -4,17 +4,19 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubeflow/mpi-operator)](https://goreportcard.com/report/github.com/kubeflow/mpi-operator)
 
 
-The MPI Operator makes it easy to run allreduce-style distributed training on Kubernetes. Please check out the list of adopters [here](./ADOPTERS.md).
+The MPI Operator makes it easy to run allreduce-style distributed training on Kubernetes. You can check out the [list of adopters of MPI Operator](./ADOPTERS.md).
 
 ## Installation
 
-You can deploy the operator with default settings without using kustomize by running the following from the repo:
+You can deploy the operator with default settings without using `kustomize` by running the following commands:
 
 ```shell
+git clone https://github.com/kubeflow/mpi-operator
+cd mpi-operator
 kubectl create -f deploy/mpi-operator.yaml
 ```
 
-Alternatively, if you haven’t already done so please follow the [Getting Started Guide](https://www.kubeflow.org/docs/started/getting-started/) to deploy Kubeflow.
+Alternatively, follow the [getting started guide](docs/started/getting-started/) to deploy Kubeflow.
```

Comment:
Ah, I think the link has to remain absolute here, as the mpi-operator repo doesn't have the power of Hugo and Netlify behind it as the website does.